frontend/src/metabase/actions/components/ActionForm/utils.ts

```diff
@@ -23,14 +23,8 @@ const getOptionsFromArray = (
   options: (number | string)[],
 ): ActionFormOption[] => options.map(o => ({ name: o, value: o }));
 
-const getSampleOptions = () => [
-  { name: t`Option One`, value: 1 },
-  { name: t`Option Two`, value: 2 },
-  { name: t`Option Three`, value: 3 },
-];
```

Comment:
Sample options might not be the best solution here, but the empty array is kinda confusing too

![Screen Shot 2023-03-07 at 3 00 48 PM](https://user-images.githubusercontent.com/30528226/223563476-d4f0d893-5334-443b-b7f6-15f99bf0bf2c.png)

the radio is even more of a "what am I supposed to do here" ?

![Screen Shot 2023-03-07 at 3 24 07 PM](https://user-images.githubusercontent.com/30528226/223567469-5a497a3b-f29b-4e3e-a91a-785c7e813752.png)

What about a single dummy option that says "Add options to this field" or some other sort of empty state prompt?

Reply:
Oh, I see, thanks for pointing me at this! I've returned the sample options back in [this commit](https://github.com/metabase/metabase/pull/29000/commits/90f047e68f8ad10996ed949c2edd390d08344ffc), but made it use `[ 1, 2, 3 ]` for "number" field type and `[ "Option One", "Option Two", "Option Three" ]` for "string" field type. What do you think?